ymchat-react-native.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   s.swift_version = "5.0"
 
   s.dependency "React"
-  s.dependency "YMChat", "~> 1.4.0"
+  s.dependency "YMChat", "~> 1.4"
```

Comment:
Looks good. This ensures that we dont have to update this line until we release 2.0
@PurushYM

Reply:
@kaunteya  or 1.5.x right?